resources/syntax/tests/syntax_test_css.css.twig

```diff
@@ -0,0 +1,22 @@
+/* SYNTAX TEST "Packages/Twig/resources/syntax/CSS (Twig).sublime-syntax" */
+
+/* Expression */
+   div {
+/* ^^^ meta.selector.css */
+/*     ^ punctuation.section.block.begin.css */
+       color: {{ blue|escape }} {# a comment #};
+/*            ^^^^^^^^^^^^^^^^^ meta.expression.twig */
+/*            ^^^^^^^^^^^^^^^^^ - meta.property-list.css */
+/*            ^^^^^^^^^^^^^^^^^ - meta.property-value.css */
```

Comment:
Not caused by this PR, but clearing all scopes is not ideal.

Reply:
Agreed, but I'm not sure of a better way to handle some of the conflicts when embedding at this point.

Reply:
The common approach would be to clear `string` scopes only, as they are treated special by various core functions. Most core syntaxes are designed to support that strategy.

Reply:
Interesting, can you link to an example of that?

Reply:
ASP, Blade, Liquid, HAML, Jinja, JSP, PHP, Slim, Svelte, Vue from the top of my head.

All of them, include 2 types of contexts into CSS/HTML/JSON/JavaScript, an "embedded" and an "interpolations" syntax both of which are likely to differ only by the latter clearing a single scope.

Reply:
Ok, I see what you're talking about. This is a templating language, so it is embedded and not interpolation with a string.

Reply:
Note scope name `embedded` being just an arbritary choise which was choosen by PHP originally long ago.

Reply:
Whatever your preferred name for them is, they are two distinct and separate concepts.